src/com/lftechnology/java/training/dipak/employeemanagement/ui/AdminPanel.java

```diff
@@ -0,0 +1,92 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.ui;
+
+import java.util.Scanner;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.LoggerFormatter;
+import com.lftechnology.java.training.dipak.employeemanagement.controller.EmployeeController;
+import com.lftechnology.java.training.dipak.employeemanagement.controller.LogOutController;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.UserType;
+
+/**
+ * <p>
+ * This class contains method that contains options for the admin access.
+ * </p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class AdminPanel {
+
+    private static final Logger LOGGER = Logger.getLogger(AdminPanel.class.getName());
+
+    static {
+        ConsoleHandler ch = new ConsoleHandler();
+        ch.setLevel(Level.FINEST);
+        LOGGER.addHandler(ch);
+        LOGGER.setUseParentHandlers(false);
+
+        LoggerFormatter myFormat = new LoggerFormatter();
+
+        ch.setFormatter(myFormat);
+    }
+
+    /**
+     * <p>
+     * This method allows the admin to choose the operations that can be performed by an admin.
+     * </p>
+     * 
+     * @param employee
+     * @param sc
+     * @throws Throwable
+     */
+    public void displayPanel(Employee employee1, Scanner sc) throws Throwable {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)